### ISSUE TITLE
Status of MLDataUtils porting

### ISSUE THREAD

**Issue by CarloLucibello** [JuliaML/MLUtils.jl#17]
### TO PORT
- [x] `noisy_sin`  -> now `Datasets.make_sin` #19
- [x] `noisy_poly`
- [x] `noisy_spiral`
- [x] `load_iris` #18 

### NOT TO PORT

- `predict, predict!` (this interface exists already in StatsAPI)
- `fit` (this interface exists already in StatsAPI)
-  `noisy_function` 
-  `expand_poly` (rethink this based on scikitlearn's [PolynomialFeatures](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.PolynomialFeatures.html#sklearn.preprocessing.PolynomialFeatures))
-  `center!, rescale!, FeatureNormalizer`. For the time being `normalise` seems enough.
-  `load_line, load_sin, load_poly, load_spiral`. These can be generated on the fly with `make_*`.

**Comment by darsnack** [comment]
I wonder if the datasets should be in MLDatasets.jl

**Comment by CarloLucibello** [comment]
I kept only iris and some toy generators, they could be useful for demonstrating functionalities without loading other libraries.
